velox/type/Type.h

```diff
@@ -662,7 +663,7 @@ class DecimalType : public ScalarType<KIND> {
   }
 
   std::string toString() const override {
-    return fmt::format("DECIMAL({},{})", precision(), scale());
+    return fmt::format("DECIMAL({}, {})", precision(), scale());
```

Comment:
This change seems unrelated and causes lots of other unrelated changes making this PR larger than needed. It looks like I misunderstood your earlier question as I didn't realize the change will be wide spread. Can it be extracted into a separate PR?